src/plugins/intel_gpu/include/intel_gpu/op/swiglu.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace intel_gpu {
+namespace op {
+
+/// \brief Operator performing Swish Gated Linear Unit Activation
+/// This operation performs gated linear unit activation that combines swish activation function
+class SwiGLU : public ov::op::Op {
+public:
+    OPENVINO_OP("SwiGLU", "gpu_opset");
+
+    SwiGLU() = default;
+    /// \brief Constructs an SwiGLU operation.
+    ///
+    /// \param data Input tensor with data
+    /// \param axis The index of an axis in "data" along which to perform the split
+    /// \param split_lenghts A list containing the sizes of each output tensor along the split "axis"
+    /// \param output_type Output element type
+    SwiGLU(const Output<Node>& data,
+           const std::vector<int64_t>& axis,
```

Comment:
As I can see, the kernel uses single axis value only. So is there a need to have a vector here?

Reply:
changed to use single axis value at primitive level